applications/asset_tracker/src/main.c

```diff
@@ -2095,13 +2152,54 @@ static void lte_handler(const struct lte_lc_evt *const evt)
 	case LTE_LC_EVT_CELL_UPDATE:
 		LOG_INF("LTE cell changed: Cell ID: %d, Tracking area: %d",
 			evt->cell.id, evt->cell.tac);
-#if defined(CONFIG_NRF_CLOUD_CELL_POS)
+#if defined(CONFIG_NRF_CLOUD_CELL_POS) && !defined(CONFIG_CELL_POS_MULTICELL)
 		/* Request location based on new network info */
 		if (data_send_enabled() && evt->cell.id != -1) {
 			k_work_reschedule_for_queue(&application_work_q,
 						       &send_cell_pos_request_work,
 						       K_SECONDS(1));
 		}
+#endif
+		break;
+	case LTE_LC_EVT_NEIGHBOR_CELL_MEAS:
+		LOG_INF("Neighbor cell measurements received: %d",
+			evt->cells_info.ncells_count);
+#if defined(CONFIG_CELL_POS_MULTICELL)
+		/* change the LED color based on the number of neighbors being used */
+		if (evt->cells_info.ncells_count > 1) {
```

Comment:
did you intend to check in this LED change?

Reply:
Probably.  I'm guessing you're hinting that I should create unique names for these colors and not reuse the GPS ones.

Reply:
i wasn't, i just thought it was something you were using for testing.
the readme will need updated to include these new UI changes.

Reply:
OK, I'll do that tomorrow.